### ISSUE TITLE
Any plans to support the new Swift build system?

### ISSUE THREAD

**Issue by BlakeLucchesi** [izqui/Taylor#36]
It looks like it would be quite trivial aside from the SwiftSockets dependency.  Has the team put any thoughts towards doing this?  I'd be happy to help contribute some effort.

**Comment by Danappelxx** [comment]
Of course! It should be very trivial - we just need to submit a PR to SwiftSockets to make it SPM compatible, and we'll do the same here. Unfortunately, it will still only run on OSX because of the Foundation dependency, but I reopened #13 and put my thoughts about that there.
